wa/workloads/googlemaps/__init__.py

```diff
@@ -0,0 +1,73 @@
+from wa import Parameter, ApkUiautoWorkload
+
+
+class GoogleMaps(ApkUiautoWorkload):
+
+    name = 'googlemaps'
+    description = '''
+    A workload to perform standard navigation tasks with Google Maps. This workload searches
+    for known locations, pans and zooms around the map, and follows driving directions
+    along a route.
+
+    To run the workload in offline mode, 'databases.tar' and 'files.tar' archives are required.
```

Comment:
Use monospace formatting (` `` `) for file names and paths.

Reply:
Sure, can do. FWIW I've used single quotes for filenames and path in the Gmail and Chrome workloads too so might want to update things there.

Reply:
Yup, I've missed those when reviewing. I've already submitted a fix for `chrome`.